common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -185,7 +185,7 @@
                                     classes = Seq("responsive-img "),
                                     widths = item.mediaWidthsByBreakpoint,
                                     maybePath = Some(imageElement.url),
-                                    maybeSrc = if(containerIndex == 0 && index < 9) Some(imageElement.url) else None,
+                                    maybeSrc = if(containerIndex == 0 && index < 4) Some(imageElement.url) else None,
```

Comment:
Just a thought on how this might be made more clear is in the naming, rather than `index` we could have something like `cardIndex` as we are a few loops in at this point.

Reply:
Good point, but can be done in a subsequent PR.